lib/identity_cache/configuration_dsl.rb

```diff
@@ -279,7 +279,7 @@ def attribute_dynamic_fetcher(attribute, fields, values) #:nodoc:
       end
 
       def add_parent_expiry_hook(options)
-        child_class = options[:association_class]
+        child_class = options[:association_class].klass
```

Comment:
True but maybe there is a way to make identity_cache autoload these classes? How about putting `require options[:association_class].class_name.lower` or similar?